### PR TITLE
Sharma 2021b added

### PR DESCRIPTION
**Pull request by TGouldBates**

### CODE REVIEW

File: sharma_2021b/odorless.csv [hunk 1]
```diff
@@ -0,0 +1,1125 @@
+Sr.No.,Name,SMILES,CAS No,Mol. Wt.,Mol. Formula,PubChem,ZINC
```
[the remark this answers is not in the document]
Comment (rgerkin) [reply sharma_2021b/odorless.csv:1]: Also "PubChem" column name should be changed to match what it is in molecules.csv -- the idea is to be able to link columns by name programatically.

**Review by rgerkin — CHANGES_REQUESTED** [review]
Unclear how index column in behavior is related to column(s) in odorants and odorless.

File: sharma_2021b/manifest.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Manifest for OlfactionBase dataset
+[source]
+doi = "https://doi.org/10.1093/nar/gkab763"
+title = "OlfactionBase: a repository to explore odors, odorants, olfactory receptors and odorant–receptor interactions."
+authors = "A.Sharma, B. K. Saha, R. Kumar, P. K. Varadwaj"
+extra = "Originally compiled by Sharma et al. 2021 and organized as OlfactionBase for public use; Pyrfume developers make no claims concerning copyright"
+
+[raw]
+raw/*.xlsx = "Odor data as initially downloaded from OlfactionBase, each file corresponds to one of 106 primary odors."
+ordors.csv = "Full dataset of odor descriptors, compliled from all files in raw/."
```
[the remark this answers is not in the document]
Comment (TGouldBates) [reply sharma_2021b/manifest.toml:10]: I must be dense, but I'm not seeing the typo in that line...

[the remark this answers is not in the document]
Comment (rgerkin) [reply sharma_2021b/manifest.toml:10]: My bad, it's the next line (ordors).

File: sharma_2021b/odorless.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (TGouldBates) [reply sharma_2021b/odorless.csv:1]: After clarifying that odorants.csv and odorless.csv are "raw" data, I'm assuming that you don't want me to change the column headings from the original, but let me know if you still do.

**Pull request merged**